src/pages/functional-testing-framework/test/action-groups.md

```diff
@@ -0,0 +1,315 @@
+    <see selector="{{AdminHeaderSection.pageTitle}}" userInput="New Category" stepKey="seeCategoryPageTitle"/>
+    <fillField selector="{{AdminCategoryBasicFieldSection.CategoryNameInput}}" userInput="{{categoryEntity.name}}" stepKey="enterCategoryName"/>
+    <click selector="{{AdminCategorySEOSection.SectionHeader}}" stepKey="openSEO"/>
+    <fillField selector="{{AdminCategorySEOSection.UrlKeyInput}}" userInput="{{categoryEntity.name_lwr}}" stepKey="enterURLKey"/>
+    <click selector="{{AdminCategoryMainActionsSection.SaveButton}}" stepKey="saveCategory"/>
+    <seeElement selector="{{AdminCategoryMessagesSection.SuccessMessage}}" stepKey="assertSuccess"/>
+    <seeInTitle userInput="{{categoryEntity.name}}" stepKey="seeNewCategoryPageTitle"/>
+    <seeElement selector="{{AdminCategorySidebarTreeSection.categoryInTree(categoryEntity.name)}}" stepKey="seeCategoryInTree"/>
+</actionGroup>
+```
+
+It can be reworked into more manageable pieces, as below. These smaller steps are easier to read, update, and reuse.
+
+- GoToCategoryGridAndAddNewCategory
+
+    ```xml
+    <actionGroup name="GoToCategoryGridAndAddNewCategory">
+        <seeInCurrentUrl url="{{AdminCategoryPage.url}}" stepKey="seeOnCategoryPage"/>
+        <click selector="{{AdminCategorySidebarActionSection.AddSubcategoryButton}}" stepKey="clickOnAddSubCategory"/>
+        <see selector="{{AdminHeaderSection.pageTitle}}" userInput="New Category" stepKey="seeCategoryPageTitle"/>
+    </actionGroup>
+    ```
+
+- FillInBasicCategoryFields
+
+    ```xml
+    <actionGroup name="FillInBasicCategoryFields">
+        <arguments>
+            <argument name="categoryEntity" defaultValue="_defaultCategory"/>
+        </arguments>
+        <fillField selector="{{AdminCategoryBasicFieldSection.CategoryNameInput}}" userInput="{{categoryEntity.name}}" stepKey="enterCategoryName"/>
+        <click selector="{{AdminCategorySEOSection.SectionHeader}}" stepKey="openSEO"/>
+        <fillField selector="{{AdminCategorySEOSection.UrlKeyInput}}" userInput="{{categoryEntity.name_lwr}}" stepKey="enterURLKey"/>
+    </actionGroup>
+    ```
+
+- SaveAndVerifyCategoryCreation
+
+    ```xml
+    <actionGroup name="SaveAndVerifyCategoryCreation">
+        <click selector="{{AdminCategoryMainActionsSection.SaveButton}}" stepKey="saveCategory"/>
+        <seeElement selector="{{AdminCategoryMessagesSection.SuccessMessage}}" stepKey="assertSuccess"/>
+        <seeInTitle userInput="{{categoryEntity.name}}" stepKey="seeNewCategoryPageTitle"/>
+        <seeElement selector="{{AdminCategorySidebarTreeSection.categoryInTree(categoryEntity.name)}}" stepKey="seeCategoryInTree"/>
+    </actionGroup>
+    ```
+
+## Elements reference
+
+### actionGroups
+
+The `<actionGroups>` element is a root element that contains XML configuration attributes.
+
+Attribute|Value|Description
+---|---|---
+`xmlns:xsi`|`"http://www.w3.org/2001/XMLSchema-instance"`|Tells the XML parser to validate this document against a schema.
```

Comment:
This table bleeds into the right margin. Maybe try removing the backticks from the Value column entries?

Reply:
That's a known issue that the devsite team is planning to fix in the theme soon. Unfortunately, that's how it's going to be until then.